packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2710,49 +2710,88 @@ class _Location {
   }
 }
 
-/// Describer to gather important information about [offendingContext].
+/// Transformer to parse and gather information about [offendingContext].
 ///
-/// This describer is only applicable when the [offendingContext] is created in
-/// widget layer.
+/// This transformer only responds to [DiagnosticsProperty] with the name
+/// 'offendingContext'.
 ///
-/// This function will be registered to [FlutterErrorDetails.offendingContextDescribers]
+/// This function will be registered to [FlutterErrorDetails.informationTransformers]
 /// in [WidgetsBinding.initInstances].
-Iterable<DiagnosticsNode> describeOffendingContext(dynamic offendingContext) {
+DiagnosticPropertiesBuilder transformOffendingContext(DiagnosticPropertiesBuilder properties) {
+  final DiagnosticPropertiesBuilder transformed = DiagnosticPropertiesBuilder();
+  bool foundStackTrace = false;
+  final List<DiagnosticsNode> afterStackTrace = <DiagnosticsNode>[];
+  // The below for loop try to achieve the following:
+  // 1. Any offendingContext found before the stacktrace stay where it is.
+  // 2. Any offendingContext found after the stacktrace move to the front of stacktrace.
+  for (DiagnosticsNode node in properties.properties) {
+    if (!foundStackTrace && node.runtimeType == DiagnosticsStackTrace)
```

Comment:
simpler is node is DiagnosticsStackTrace 
instead of
node.runtimeType == DiagnosticsStackTrace.
The only case to use 
node.rumtimeType == Type 
is when you want to ensure you get the act type and not a subclass.